pkg/cloud/provider/csvprovider.go

```diff
@@ -178,18 +183,22 @@ func (c *CSVProvider) DownloadPricingData() error {
 		} else if p.AssetClass == "gpu" {
 			gpupricing[key] = &p
 			c.GPUMapFields = append(c.GPUMapFields, strings.ToLower(p.InstanceIDField))
+		} else if p.AssetClass == "gpulabel" {
+			labelKeyValue := p.InstanceIDField + "=" + p.InstanceID
```

Comment:
Here, `InstanceIDField` represents the label name, and `InstanceID` represents the label value. So, any instance of `labelName=labelValue` on nodes with GPUs will match this filter.